plugins/Feedback/API.php

```diff
@@ -27,15 +28,20 @@ class API extends \Piwik\Plugin\API
      * Sends feedback for a specific feature to the Matomo team or alternatively to the email address configured in the
      * config: "feedback_email_address".
      *
-     * @param string      $featureName  The name of a feature you want to give feedback to.
-     * @param bool|int    $like         Whether you like the feature or not
-     * @param string|bool $message      A message containing the actual feedback
+     * @param string|null $featureName  The name of a feature you want to give feedback to.
+     * @param int         $like         Whether you like the feature or not
+     * @param string|null $choice       Multiple choice option chosen
+     * @param string|null $message      A message containing the actual feedback
      */
-    public function sendFeedbackForFeature($featureName, $like, $message = false)
+    public function sendFeedbackForFeature(?string $featureName, int $like, ?string $choice, ?string $message = null)
     {
         Piwik::checkUserIsNotAnonymous();
         Piwik::checkUserHasSomeViewAccess();
 
+        if (strlen($message) < 4) {
```

Comment:
`$message` might be `null`. Might be good to check if it's not empty as well. I'm actually no sure if `strlen(null)` might not trigger a warning on PHP 8.